curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6554ac937a49be2701af4f2f.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 6554ac937a49be2701af4f2f
-title: Step 61
+title: Step 65
 challengeType: 20
-dashedName: step-61
+dashedName: step-65
 ---
 
 # --description--
```

Comment:
Above the `offset` variable, create another comment saying `Define the offset and the encrypted letter`.